__tests__/sbom/main-test-app/verify-sbom.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+cd target
```

Comment:
I'm not a big fan of scripts that need to be run from a specific directory. I guess you could do this:

```suggestion
script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```